axlearn/common/conformer.py

```diff
@@ -202,6 +202,11 @@ class Config(BaseLayer.Config):
         )
         lconv: LConvLayer.Config = LConvLayer.default_config()
         norm: LayerNorm.Config = LayerNorm.default_config()
+        # Layer order. If None, default to "mhsa_before_conv", i.e., conformer layer order as
+        # secified in https://arxiv.org/abs/2005.08100.
+        # If not None, only "lconv_before_ff" "lconv_before_mhsa" "mhsa_before_lconv" is allowed.
```

Comment:
```suggestion
        # If not None, only "lconv_before_ff" "lconv_before_mhsa" "mhsa_before_lconv" are allowed.
```

You can also consider a Literal[...] typing.

Reply:
Thanks for the suggestion. Updated.